plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/azure.ts

```diff
@@ -44,6 +44,7 @@ export function createPublishAzureAction(options: {
     gitAuthorName?: string;
     gitAuthorEmail?: string;
   }>({
+    supportsDryRun: true,
```

Comment:
Not sure that this is true? It's a mutating action, so it doesn't support dry running.